docs/SUMMARY.md

```diff
@@ -41,6 +41,21 @@
   * [Progressbar](execution-analysis/progressbar.md)
   * [Formatting and drawing](execution-analysis/formatting_and_drawing.md)
 
+## Performance Tips
```

Comment:
Sorry @umut-sahin: It's not ok to add a new section in the SUMMARY. The section org is managed by @yuxizama (with me as a consultant!). 

Also, looking at the content and the style, it is:
- a bit too complex for "basic part"
- non actionable several times, it's more explanations
so what about moving it to Explanations, and having a **small** and **actionable** (which options should the user set, which trick should she try) in the basic part?

```
index bb5512a3..e69b9b8d 100644
--- a/docs/SUMMARY.md
+++ b/docs/SUMMARY.md
@@ -30,6 +30,7 @@
 * [Compression](compilation/compression.md)
 * [Reusing arguments](compilation/reuse_arguments.md)
 * [Common errors](compilation/common_errors.md)
+* [Performance Tips][some_short_markdown.md]
 
 ## Execution / Analysis
 
@@ -71,6 +72,19 @@
   * [Bitwise operations](core-features/bitwise.md)
   * [Direct circuits](compilation/direct_circuits.md)
   * [Tagging](core-features/tagging.md)
+* [Advanced performance tips][some_markdown.md]
+  * [Improve parallelism](optimization/improve-parallelism/self.md)
+    * [By enabling dataflow parallelism](optimization/improve-parallelism/dataflow.md)
+    * [By tensorizing operations](optimization/improve-parallelism/tensorization.md)
+  * [Optimize table lookups](optimization/optimize-table-lookups/self.md)
+    * [By reducing the amount of table lookups](optimization/optimize-table-lookups/reducing-amount.md)
+    * [By changing the implementation strategy of complex operations](optimization/optimize-table-lookups/strategies.md)
+    * [By using round/truncate bit pattern before table lookups](optimization/optimize-table-lookups/round-truncate.md)
+    * [By activating approximate mode for rounding](optimization/optimize-table-lookups/approximate.md)
+    * [By utilizing bit extraction](optimization/optimize-table-lookups/bit-extraction.md)
+  * [Optimize cryptographic parameters](optimization/optimize-cryptographic-parameters/self.md)
+    * [By adjusting table lookup error probability](optimization/optimize-cryptographic-parameters/p-error.md)
+    * [By specifying composition when using modules](optimization/optimize-cryptographic-parameters/composition.md)
 * [Cryptography basics](core-features/fhe_basics.md)
```

Before doing it, let's validate with @yuxizama

Reply:
We've had this discussion offline